File: code/drasil-example/Drasil/Projectile/src/OutputFormat.py
from __future__ import print_function
import sys
import math


def write_output(message, offset):
    outfile = open("output.txt", "w")
    print(message, file=outfile)
    print("Offset: ", end='', file=outfile)
    print(offset, file=outfile)
    outfile.close()

File: code/stable/projectile/src/cpp/OutputFormat.cpp
#include "OutputFormat.hpp"

#include <algorithm>
#include <iostream>
#include <fstream>
#include <iterator>
#include <string>
#include <math.h>
#include <sstream>
#include <limits>
#include <vector>

using std::string;
using std::vector;
using std::ifstream;
using std::ofstream;

void write_output(string s, double d_offset) {
    ofstream outputfile;
    outputfile.open("output.txt", std::fstream::out);
    outputfile << "s = ";
    outputfile << s << std::endl;
    outputfile << "d_offset = ";
    outputfile << d_offset << std::endl;
    outputfile.close();
}

